subsys/testsuite/include/busy_sim.h

```diff
@@ -6,6 +6,8 @@
 #ifndef BUSY_SIM_H__
 #define BUSY_SIM_H__
 
+typedef void (*busy_sim_cb_t)(void);
```

Comment:
Callback typedef without a `void*` argument? Always a good sign that another change will be coming to add it soon ;)

Reply:
I hope not as long it is used in testing environment.